Automattic/simplenote-ios
# Apply minimum cell height in condensed list mode

### Fix
In this PR we're applying minimum height of 44pt to the note list cells in condensed mode.
In addition this PR fixes incorrect separator margins. This issue happens for reused cells when note list is scrolled.

Closes #1045

|Before|After|
|-|-|
|![Simulator Screen Shot - iPhone 12 - 2021-01-12 at 13 51 07](https://user-images.githubusercontent.com/54751/104319900-d736ce00-54e1-11eb-88d1-96e98efd3588.png)|![Simulator Screen Shot - iPhone 12 - 2021-01-12 at 16 11 43](https://user-images.githubusercontent.com/54751/104332976-043ead00-54f1-11eb-95de-8217d3abff85.png)|

### Test
1. Enable condensed mode
2. Open note list

- [ ] Check the cells height is at least 44pt
- [ ] Scroll the list back and forth. Check that separators are always aligned with the text

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 1a79bb2b with:
> Updated minimum row height when note list is in condensed mode #1045
